### ISSUE TITLE
New release with parking_lot 0.8 

### ISSUE THREAD

**Issue by atouchet** [rust-windowing/glutin#1198]
Would it be possible to have a 0.21.1 release with parking_lot updated to 0.8 (1d93aae3507f52f91981ad5799e86d31617a69a3)? winit had a 0.19.2 release which included that update and bumping parking_lot in glutin as well would help remove duplicate dependencies.

**Comment by est31** [comment]
Now we have 0.9.

**Comment by atouchet** [comment]
The latest stable release of winit still appears to be using 0.8.

**Comment by ZeGentzy** [comment]
I missed this issue, apologies. Yeah, a new release would be fine by me, just open a PR.

**Comment by est31** [comment]
@atouchet yes but it's better to update to the very newest release. Winit should do it too: https://github.com/rust-windowing/winit/issues/1097

**Comment by atouchet** [comment]
@ZeGentzy Is there a branch for glutin 0.21.x?

@est31 0.8 is fine for now for my purposes. This is mostly just to eliminate the duplicate dependency.

**Comment by ZeGentzy** [comment]
I've made a branch called `backport21`, you're PR is going to have to modify travis ever so slightly for it to push. Do something similar to like what I did to the backports branch here: https://github.com/rust-windowing/glutin/commit/9cac705d08dd5294dd6a6042217e659696fbed46#diff-354f30a63fb0907d4ad57269548329e3R68

I'd rather the PR bumps it to version 0.9, instead of 0.8.

**Comment by atouchet** [comment]
If it gets bumped to 0.9 winit will need another release as well. I think @Eijebong has been working on updating dependencies in Servo, perhaps he has an opinion on this.

**Comment by est31** [comment]
@atouchet winit is right now working on a new semver breaking release (0.20.0). I don't know how long it'll take until it's out. Maybe the patch to update parking lot can be backported onto their `winit-legacy` branch.

**Comment by atouchet** [comment]
https://github.com/rust-windowing/winit/pull/1109
